src/gui_tool/entry.py

```diff
@@ -50,31 +50,24 @@ def split_file(file_loc, mdi:MetaDataItem):
             context = GUIContext(file_loc)
             gui = SPGUIManager(context)
             rs = gui.start()
-            ret = []
 
-            split_vid = False
-            if len(rs) > 1:
```

Comment:
If len(rs) == 1, we will still be setting the video as is_split_url=True even though the URL will still be unique.